mlflow/server/auth/sqlalchemy_store.py

```diff
@@ -259,3 +265,92 @@ def rename_registered_model_permissions(self, old_name: str, new_name: str):
             )
             for perm in perms:
                 perm.name = new_name
+
+    def create_scorer_permission(
+        self, experiment_id: str, scorer_name: str, username: str, permission: str
+    ) -> ScorerPermission:
+        _validate_permission(permission)
+        with self.ManagedSessionMaker() as session:
+            try:
+                user = self._get_user(session, username=username)
+                perm = SqlScorerPermission(
+                    experiment_id=experiment_id,
+                    scorer_name=scorer_name,
+                    user_id=user.id,
```

Comment:
Q:
are we going to support granting permission to a group of users (e.g. I want to grant permission to **all users** for a scorer, but I don't want to add `SqlScorerPermission` item for every user, this way needs updates if new users are created.

Reply:
RBAC sounds very cool, but our auth system doesn't support it for any existing entities. Do I think we should eventually support it? YES. But I think that should be part of the 'bring auth directly into the tracking server as a core feature' sort of work. I do think we need this. Just not for this PR.